apiserver/facades/client/application/deploy.go

```diff
@@ -70,10 +71,16 @@ type MachineSaver interface {
 	Save(context.Context, string) error
 }
 
+// ApplicationSaver instances save an application to dqlite state.
+type ApplicationSaver interface {
+	Save(ctx context.Context, name string, units ...applicationservice.AddUnitParams) error
```

Comment:
It's been noted that these should be `SaveApplication`. We're getting rid of just verbs for methods.